src/osd/PeeringState.cc

```diff
@@ -4092,7 +4092,7 @@ void PeeringState::calc_trim_to_aggressive()
 		 PG_STATE_BACKFILLING |
 		 PG_STATE_BACKFILL_WAIT |
 		 PG_STATE_BACKFILL_TOOFULL)) {
-    target = cct->_conf->osd_max_pg_log_entries;
+    target = pl->get_target_pg_log_entries();
```

Comment:
I missed this earlier - I don't think we should change our memory allocation based on whether we're in recovery - that was a conscious choice when we set both min and max to the same with the pg log hard limit.

We don't necessarily always have memory autotuning, and if we do it's still a big chunk out of the cache.

I'd suggest unconditionally using get_target_pg_log_entries() as the target.